src/AppBundle/AppBundle.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /*
```

Comment:
this is the only license header I could find in the project code. Should we add it to the other classes as well?